src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -991,6 +989,22 @@ private function getKernelRootHash(ContainerBuilder $container)
     }
 
     /**
+     * Gets the path of a given bundle using Bundle#getPath() when available.
+     *
+     * @param string $fqcn The bundle FQCN
+     *
+     * @return string
+     */
+    private function getBundlePath($fqcn)
+    {
+        $reflection = new \ReflectionClass($fqcn);
+        $getPathReflection = new \ReflectionMethod($fqcn, 'getPath');
+        $reflectionInstance = $reflection->newInstanceWithoutConstructor();
```

Comment:
I don't think it's a solution, the bundle could use on of it's constructor arguments to determine it's path.